src/NServiceBus.Core/Performance/Metrics/MetricsExtensions.cs

```diff
@@ -0,0 +1,52 @@
+namespace NServiceBus.Performance.Metrics
+{
+    using System;
+    using System.Collections.Generic;
+    using Features;
+
+    /// <summary>
+    /// Extension methods to configure OpenTelemetry metrics.
+    /// </summary>
+    public static class MetricsExtensions
+    {
+        /// <summary>
+        /// Configures NServiceBus to report messaging metrics using OpenTelemetry.
+        /// </summary>
+        /// <param name="config">The <see cref="EndpointConfiguration" /> instance to apply the settings to.</param>
+        public static EndpointConfiguration EnableOpenTelemetryMetrics(this EndpointConfiguration config)
+        {
+            Guard.AgainstNull(nameof(config), config);
+            config.EnableFeature<MessagingMetricsFeature>();
+
+            return config;
+        }
+    }
+
+    static class Extensions
+    {
+        public static void ThrowIfSendOnly(this FeatureConfigurationContext context)
```

Comment:
I like this but it's very specific to this feature and it will show up in Intellisense for every other feature. Maybe we could pass in the `"Metrics"` part of the exception message to make it more generic. It might be easier to just inline it into the feature.

I suspect `ThrowIfSendOnly` probably comes up a lot in the code though so happy to see this get added somewhere.

Reply:
moved it, there were not many similar usages to warrant a reuse for now.